File: JS/Sequencial/EX6_A.js
/**
 * 6. faça um programa que receba o salario-base de um funcionario, calcule e mostre o salario a receber, sabendo-se que esse funcionario 
 * em gratificação de 5% sobre o salario base e paga imposto de 7% sobre o salario-base.
 */

confirm("Calcular salario base com 5% de  gratificação e 7% de imposto ")

var sal_base = prompt("Adicione o salário")
sal_base = parseFloat(sal_base)

var imp = sal_base * 7/100;
var grat = sal_base * 5/100;

sal_atual = sal_base + grat - imp

console.log("O calculo do imposto sobre o salario é: "+ imp +" - "+ sal_base)
console.log("O calculo da gratificação sobre o salario é:  "+ grat +" + "+ sal_base)

console.log("O salario atual é: "+ sal_atual)

File: python/Sequencial/EX6_A.py
#6. faça um programa que receba o salario-base de um funcionario, calcule e mostre o salario a receber,
# sabendo-se que esse funcionario em gratificação de 5% sobre o salario base e paga
# imposto de 7% sobre o salario-base.

print("***********************SEQUENCIAL***************************")
print("************************Programa*6**************************")
print("*Calcula salario base com gratificação de 5%e imposto de 7%*")
print("************************************************************")

salario_base = float(input("Digite o salario: "))

grat = salario_base * 5/100
imp  = salario_base * 7/100

salario_atual = salario_base + grat - imp

print("O Salario base é R${: .2f} ".format(salario_base))
print("A gratificação é R${: .2f} | O imposto é R${: .2f}".format(grat,imp))
print("Salário atual R${: .2f}".format(salario_atual))